src/Symfony/Component/Routing/Loader/AnnotationFileLoader.php

```diff
@@ -92,6 +92,11 @@ protected function findClass($file)
         $class = false;
         $namespace = false;
         $tokens = token_get_all(file_get_contents($file));
+
+        if (1 === count($tokens) && T_INLINE_HTML === $tokens[0][0]) {
```

Comment:
I think so. We're catching the case of forgetting the open PHP tag here, not whether the PHP tag is the first token in the file.